proto/api.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+package cyneruxyz.api.v1;
+option go_package = "./proto";
+
+import "google/api/annotations.proto";
+
+
+service APIService {
+  rpc Echo(EchoRequest) returns(Response) {
+    option (google.api.http) = {
+      post:  "/echo"
+    };
+  }
+  rpc CreateAddressField(AddressFieldRequest) returns (AddressField) {
+    option (google.api.http) = {
+      post: "/addressbook"
+      body: "*"
+    };
+  }
+
+  rpc ReadAddressField(AddressFieldQuery) returns (stream AddressField) {
+    option (google.api.http) = {
+      get:  "/addressbook"
+    };
+  }
+
+  rpc UpdateAddressField(AddressFieldUpdateRequest) returns (Response) {
+    option (google.api.http) = {
+      put:  "/addressbook"
+      body: "*"
+    };
+  }
+  rpc DeleteAddressField(AddressFieldRequest) returns (Response) {
+    option (google.api.http) = {
+      delete: "/addressbook"
+    };
+  }
```

Comment:
What are we going to delete here? `body` can be used in DELETE, but it can be done more gracefully.